modules/auxiliary/spoof/mdns/mdns_response.rb

```diff
@@ -247,6 +247,7 @@ def cleanup
       self.thread.kill
       self.thread = nil
     end
+    self.sock.close
```

Comment:
Given these there may be some issues with a call to `close` if the service was started by `service_manager`.

Reply:
As far as I can tell, the service manager interacts with `Rex::Proto::XXX::Server` objects; or more specifically, objects that have a `start` and a `stop` method. As these responder classes are modules, not servers, I don't know if it would be even possible for the `ServiceManager` to launch them.